INSTALL.txt

```diff
@@ -54,6 +54,27 @@ currently support.  You can download it from:
     https://github.com/libyal/libewf
     Available at: http://sourceforge.net/projects/libewf/
 
+- LibVHDI: Allows you to process disk images that are stored in the
```

Comment:
technically the project name is libvhdi (without the casing)

Reply:
Changed.